guides/source/active_record_validations.md

```diff
@@ -799,9 +799,9 @@ class Person < ApplicationRecord
   # Hard-coded message
   validates :name, presence: { message: "must be given please" }
 
-  # Message with dynamic attribute value. %{value} will be replaced with
-  # the actual value of the attribute. %{attribute} and %{model} also
-  # available.
+  # Message with dynamic attribute value. %{value} will be replaced
+  # with the actual value of the attribute. %{attribute} and %{model}
```

Comment:
Why wrap "with" onto the next line here?

Reply:
It is rendered on the next line in the guides: ![Active+Record+Validations+%E2%80%94+Ruby+on+Rails+Guides+2020-07-13+09-33-25](https://user-images.githubusercontent.com/16672/87281341-693e2980-c4f3-11ea-895b-b047f255d8fa.png)